rasterio/windows.py

```diff
@@ -30,6 +42,15 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def toranges(window):
```

Comment:
It makes sense, but I think it'll require more code. I'd like to leave this function in for the time being.